src/formatters/factory.ts

```diff
@@ -2,11 +2,12 @@ import { Formatter } from "./index";
 import { Csv } from "./csv";
 import { Json } from "./json";
 import { Text } from "./text";
+import { Xunit } from "./xunit";
 import { Formatter as FormatterName } from "../enumerations";
 
 export class Factory {
     static getInstance(format: FormatterName): Formatter {
-        const classes = { Csv, Json, Text };
+        const classes = { Csv, Json, Text, Xunit };
```

Comment:
Missing UT in /home/teo/projects/temp/license-compliance/tests/formatters/factory.spec.ts